### ISSUE TITLE
Scaffold-DbContext with create default

### ISSUE THREAD

**Issue by reader-man** [aspnet/EntityFramework#8077]
Describe what is not working as expected.
When running Scaffold-DbContext
  The Generation of Fluent API for the default value in Context.OnModelCreating(ModelBuilder modelBuilder), returns SQL Server comments which affects the compilation of the Project in VS.

If you are seeing an exception, include the full exceptions details (message and stack trace).

```
Compilation Error in intellisense.
```

### Steps to reproduce
make a new database in sql server, and run the file "sqlSampleTableToProveGenerationProblem.sql" and don't delete the first empty lines:

https://goo.gl/w1Fb5v

then generate the POCO Entities + the Context, and you will see these problematic lines:
```
                entity.Property(e => e.AnythingCol)
                    .HasColumnName("anythingCol")
                    .HasColumnType("TypeBoolean_Dom")
                    .HasDefaultValueSql("


/*==============================================================*/
/* Default: D_0                                                 */
/*==============================================================*/
create default D_0
    as 0
");
            });
```

Partial code listings, or multiple fragments of code, will slow down our response or cause us to push the issue back to you to provide code to reproduce the issue.

```c#
<code listing>
```

### Further technical details
EF Core version: 1.1
Database Provider: Microsoft.EntityFrameworkCore.SqlServer
Operating system: Win 10
IDE: Visual Studio 2017

**Comment by bricelam** [comment]
Note to fixer, Migrations [handles this](https://github.com/aspnet/EntityFramework/blob/dev/src/EFCore.Design/Migrations/Design/CSharpHelper.cs#L292-L295) by generating a verbatim string literals for multi-line strings.

**Comment by jnegron357** [comment]
I'm getting the same issue except my output is just a bunch of empty lines:
`.HasDefaultValueSql("create default [DEFgetdate] as getdate()






");
`
When I look at the definition for the custom default DEFgetdate I see that it also has several lines of empty space, could it be the EF core is picking it up from there? If so then maybe it just needs a Trim() function call at the time of generation of code.

We are using Sql Server 2008 R2 (SP2) - 10.50.4042.0 (X64) 

EF Core version: 1.1.1
Database Provider: Microsoft.EntityFrameworkCore.SqlServer
Operating system: Win 7
IDE: Visual Studio Professional 2017
Version 15.1 (26403.0) Release

**Comment by lajones** [comment]
Fixed by PR #8145. Note: we are just making sure the C# works here, we are not getting into trimming just yet. For now I'd rather just report exactly what SQL Server has stored rather than interpret it in any way. That may change as I think there's another bug where the default value wasn't right because we weren't correctly counting the open and close parentheses that SQL Server had stored in the default value. I can't find that bug right now, but when we address that we can think about whether to trim at the same time. On the other hand we may choose not to change it so that round tripping (DB -> Scaffolding -> updating DB via Migrations) works.
